source/Government.cpp

```diff
@@ -139,6 +139,10 @@ void Government::Load(const DataNode &node)
 			raidFleet = GameData::Fleets().Get(child.Token(1));
 		else if(child.Token(0) == "provoked on scan")
 			provokedOnScan = true;
+		else if(child.Token(0) == "show reputation")
+			showReputation = true;
+		else if(child.Token(0) == "hide reputation")
+			showReputation = false;
```

Comment:
it is hidden by default, so is there a need to have a token to toggle it off?
Same logic applies if you reverse it and show it by default.

Reply:
Event changes.